core/banking-protocol/src/test/java/de/adorsys/opba/core/protocol/controller/TestTppBankSearchController.java

```diff
@@ -0,0 +1,62 @@
+package de.adorsys.opba.core.protocol.controller;
+
+import de.adorsys.opba.core.protocol.BaseMockitoTest;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.web.context.WebApplicationContext;
+
+import static de.adorsys.opba.core.protocol.TestProfiles.ONE_TIME_POSTGRES_ON_DISK;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.jsonPath;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+import static org.springframework.test.web.servlet.setup.MockMvcBuilders.webAppContextSetup;
+
+@SpringBootTest
+@ActiveProfiles(ONE_TIME_POSTGRES_ON_DISK)
+class TestTppBankSearchController extends BaseMockitoTest {
+
+    private MockMvc mockMvc;
+
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @BeforeEach
+    void onSetUp() {
+        this.mockMvc = webAppContextSetup(webApplicationContext).build();
```

Comment:
`@AutoConfigureMockMvc` should be enough instead of `this.mockMvc = webAppContextSetup(webApplicationContext).build();`

Reply:
done